pkg/controller/common/helpers.go

```diff
@@ -210,16 +249,16 @@ func convertIgnition2to3(ign2config ign2types.Config) (ign3types.Config, error)
 	if err != nil {
 		return ign3types.Config{}, errors.Errorf("unable to convert Ignition spec v2 config to v3: %v", err)
 	}
-	// Workaround to get a v3.1 config as output
-	converted3 := translate3.Translate(ign3_0config)
+	// Workaround to get a v3.2 config as output
+	converted3 := translate3.Translate(translate3_1.Translate(ign3_0config))
```

Comment:
Actually, this would translate to 3.1 still. Did you want it to translate to 3.2?

Reply:
i read this as  3.0->3.1->3.2? am i wrong?

Reply:
Oh whoops I misread completely, ignore me. LGTM!